pkg/provider/provider.go

```diff
@@ -142,14 +146,41 @@ var (
 	loaded = false
 )
 
-func buildTestEnvironment() { //nolint:funlen
+// Build image with version based on environment variables if provided, else use a default value
```

Comment:
I'd consider:
```
func getEnv(key, fallback string) string {
    if value, ok := os.LookupEnv(key); ok {
        return value
    }
    return fallback
}
```

More details:
 https://stackoverflow.com/questions/40326540/how-to-assign-default-value-if-env-var-is-empty

Reply:
Ah yes, this is just a copy/paste from the old function.  I can fix this in a subsequent PR.

Reply:
Just kidding, I fixed it.